app/lib/core/services/navigation/nav_service.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:flutter/material.dart';
+
+class NavigationService {
```

Comment:
I like this change, abstracting out how we navigate should increase understandability and reduce our need to describe why we have/had two navigators